api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/WeightMappings.kt

```diff
@@ -211,17 +159,22 @@ internal object WeightMappings {
                 MOVING_MEAN_DATA_PATH_TEMPLATE,
                 MOVING_VARIANCE_DATA_PATH_TEMPLATE
             )
-        val variables = mutableMapOf(
-            Pair(batchNormMovingMeanVarName(layer.name), layerBatchNormPaths.movingMeanPath),
-            Pair(batchNormMovingVarianceVarName(layer.name), layerBatchNormPaths.movingVariancePath)
+        return mapOfNotNull(
+            layer.movingMean to layerBatchNormPaths.movingMeanPath,
+            layer.movingVariance to layerBatchNormPaths.movingVariancePath,
+            layer.gamma to layerBatchNormPaths.gammaPath,
+            layer.beta to layerBatchNormPaths.betaPath
         )
-        if (layer.scale) {
-            variables[batchNormGammaVarName(layer.name)] = layerBatchNormPaths.gammaPath
-        }
-        if (layer.center) {
-            variables[batchNormBetaVarName(layer.name)] = layerBatchNormPaths.betaPath
+    }
+
+    private fun <K, V> mapOfNotNull(vararg mapping: Pair<K?, V?>): Map<K, V> {
```

Comment:
Looks like we need to extend the standard library here!
Probably better to move such common things to the high-level util class at the level of project (if something common will be useful again)

Reply:
Moved the function to a utility file.